ui/perfherder/perf-helpers/helpers.js

```diff
@@ -776,6 +782,9 @@ export const createGraphData = (seriesData, alertSummaries, colors, symbols) =>
         alertSummary: alertSummaries.find(
           (item) => item.push_id === dataPoint.push_id,
         ),
+        commonAlert: otherAlerts.some((item) => {
+          return item.find((alert) => alert.revision === dataPoint.revision);
```

Comment:
I think it should be best to keep the same logic as on line `783` on finding the alerts with the same `push_id`.